pkg/cmd/alpha/alpha.go

```diff
@@ -56,7 +56,7 @@ func NewCmdAlpha(commonOpts *opts.CommonOptions) (*cobra.Command, error) {
 
 	plugins, err := options.getPluginSpecs()
 	if err != nil {
-		return cmd, err
+		log.Logger().Warnf("failed to discover alpha commands from github: %s", err.Error())
```

Comment:
should we return at this point? I've not looked at the code so am unsure at what `plugins` will be

Reply:
it looks like it could be nil

Reply:
its fine for it to be nil as iterating over a nil list is fine in go

e.g.
https://play.golang.org/p/igI2_8v9StC

if its nil it just doesn't add any sub commands

Reply:
@garethjevans any chance I can convince you to /lgtm again? :)